examples/Bunch/include/simulation_defines/param/gasConfig.param

```diff
@@ -47,6 +47,13 @@ namespace gasProfiles
 {
 
 PMACC_STRUCT(GaussianCloudParam,
+    /** Gas Formula:
+     *     exponent = |globalCellPos - center| / sigma
+     *     density = e^[ gasFactor * exponent^gasPower ]
+     */
+    (PMACC_C_VALUE(float_X, gasFactor, -1.0))
```

Comment:
Could we use -1/2 as default value here as the standard [Gauss distribution](https://en.wikipedia.org/wiki/Gaussian_function)?